storybook/src/components/TextInput/TextInput.docs.mdx

```diff
@@ -14,14 +14,116 @@ import README from "../../../../packages/css/src/components/text-input/README.md
 
 ## Examples
 
+### Type: Password
+
+Creates an input where the user can enter a password.
+The characters entered are hidden, usually represented by dots or asterisks, so that the actual text is not visible.
+
+Use the password type when the input requires sensitive information, like passwords or PINs.
+It ensures that the input is not readable by others who might be looking at the screen.
+
+Guidelines:
+
+1. Allow the user’s password manager to automatically enter the password through `autocomplete="current-password"`.
+   When asking for a new password, use `autocomplete="new-password"` instead.
+2. Add a `minlength` attribute to ensure passwords meet a minimum length requirement.
+3. Use the `pattern` attribute to enforce password policies, like including numbers and special characters.
+4. If the password is a numeric PIN, add `inputmode="numeric"`.
+   Devices with virtual keyboards then switch to a numeric keypad layout which makes entering the password easier.
+
```

Comment:
We should add here that you should '[use `autocapitalize="none"`, `autocorrect="off"` and `spellcheck="false"` to stop browsers automatically changing user input on fields that expect grammatically incorrect data—like email addresses and passwords](https://adamsilver.io/blog/form-design-from-zero-to-hero-all-in-one-blog-post/)'.

Reply:
I’ve done so.